### ISSUE TITLE
User Account Settings Page needed

### ISSUE THREAD

**Issue by Jake3231** [UTDNebula/jupiter#52]
# Overview

As a user, I should be able to access settings for my Jupiter account.

We will need to start by creating a user account page in the project, and adding some skeleton UI, such as a sidebar or other internal navigation with a few options. Examples could be: General, Interests, Security.

# Proposal

- [ ] Initialize new account settings page
- [ ] Create skeleton navigation UI with a few sections that shows independent UI
- [ ] Wire up to navbar if #51 is also completed.

## Supplementary Information

This may require some design consideration also. If you're unsure about how this should look, please reach out here or in the Discord asking for ideas, and we can brainstorm different designs.

**Comment by Jake3231** [comment]
This should likely include general app settings, including account settings

**Comment by Jake3231** [comment]
Sending task for collaboration, as this could require splitting into smaller pieces

**Comment by RubenOlano** [comment]
Do we have any idea on how we're gonna implement the auth flow or how we're gonna authenticate users?

**Comment by Jake3231** [comment]
I don't think we have a solution yet for that. We may be able to implement a prototype or proof of concept using the in-development backend. We could reach out to other teams to get advice on what they've use and how we can get off the ground with a functional auth flow.

**Comment by RubenOlano** [comment]
Yeah, I think if we're able to keep in consistent across other projects then that would be good. The only solutions off the top of my head would be using something like firebase or nextauth

**Comment by Jake3231** [comment]
As we may want to wait for further discussion to actually begin implementing user auth, we could probably still make the skeleton UI for this, though it may be difficult to go too far into the app setting before some backend is ready.

**Comment by Jake3231** [comment]
@iammaxlichter
